pkg/scannerv4/mappers/mappers.go

```diff
@@ -442,6 +510,13 @@ func toProtoV4VulnerabilitiesMap(ctx context.Context, vulns map[string]*claircor
 			Cvss:               preferredCVSS,
 			CvssMetrics:        metrics,
 		}
+		if vulnEPSS != (epssDetail{}) {
+			vulnerabilities[k].EpssMetrics = &v4.VulnerabilityReport_Vulnerability_EPSS{}
+			vulnerabilities[k].EpssMetrics.ModelVersion = vulnEPSS.ModelVersion
+			vulnerabilities[k].EpssMetrics.Date = vulnEPSS.Date
+			vulnerabilities[k].EpssMetrics.Probability = float32(vulnEPSS.EPSS)
+			vulnerabilities[k].EpssMetrics.Percentile = float32(vulnEPSS.Percentile)
+		}
```

Comment:
Can we create a map function for `EpssMetrics` that returns the object so you can assigned like:

```golang
vulnerabilities[k] = &v4.VulnerabilityReport_Vulnerability{
			Id:                 v.ID,
			...
			EpssMetrics: epssMetrics(vulnEPSS)
			...
}
```